### ISSUE TITLE
An problem of reproduction

### ISSUE THREAD

**Issue by Ldj-2004** [mahaoxiang822/Generalizing-Grasp#9]
Thanks for your excellent job! But I seem to have encountered some problems when reproducing the code, the metrics I reproduce are very different from the ones in the paper,.I use the default parameters provided in the author's code throughout the process, and set the collision_thresh of test =1 when testing. The reproduction results are as follows, do you have any suggestions, thank you!@mahaoxiang822 @qingchenkanlu @lishiyu0088 @huamo555 @huamo555
Evaluation Result:
realsense, AP Seen=0.5341908802440621
seen
AP 0.5341908802440621
(30, 1, 1, 6)
AP0.4 0.4673469886628131
AP0.8 0.6318127928308598
Evaluation Result:
realsense, AP=0.5049206094190755, AP Similar=0.5049206094190755
similar
AP 0.5049206094190755
(30, 1, 1, 6)
AP0.4 0.4480599532912056
AP0.8 0.5913196706439465

Evaluation Result:
realsense, AP=0.2516279677509838, AP Novel=0.2516279677509838
novel
(30, 1, 1, 6)
AP 0.2603047942251557
AP0.4 0.14280040710553046
AP0.8 0.29526223637587584

**Comment by mahaoxiang822** [comment]
Can you close the collision detection during inference by setting the collision_thresh=0 and report the results?
It is also be helpful to offer the training Pytorch version and the GPU you use

**Comment by Ldj-2004** [comment]
Thanks for your replying.And one thing I forget to say is that the result above is without optimization.But I still feel the result is very strange from the metric given in the essay.I train the program with a single NVIDIA GeForce RTX 3090(24G) and torch                     2.1.0+cu121 @mahaoxiang822
By the way ,you can check more detail information in your email if available.My email address is 202202810616@email.sxu.edu.cn. I think you've still remember me haha~
And the reproduction results by setting the collision_thresh=0 are as follow.It seems the same as before.
Evaluation Result:
realsense, AP Seen=0.5341908802440621
seen
AP 0.5341908802440621
(30, 1, 1, 6)
AP0.4 0.4673469886628131
AP0.8 0.6318127928308598

Evaluation Result:
realsense, AP=0.5049206094190755, AP Similar=0.5049206094190755
similar
AP 0.5049206094190755
(30, 1, 1, 6)
AP0.4 0.4480599532912056
AP0.8 0.5913196706439465

Evaluation Result:
realsense, AP=0.2516279677509838, AP Novel=0.2516279677509838
novel
(30, 1, 1, 6)
AP 0.2603047942251557
AP0.4 0.14280040710553046
AP0.8 0.29526223637587584

**Comment by Ldj-2004** [comment]
When I'm training ,I also encountered some warnings which I don't know if it's matters.

/home/lidongjiang/anaconda3/envs/ldjpytorchlinux/lib/python3.9/site-packages/MinkowskiEngine-0.5.4-py3.9-linux-x86_64
.egg/MinkowskiEngine/__init__.py:36: UserWarning: The environment variable `OMP_NUM_THREADS` not set. MinkowskiEngine
 will automatically set `OMP_NUM_THREADS=16`. If you want to set `OMP_NUM_THREADS` manually, please export it on the 
command line before running a python script. e.g. `export OMP_NUM_THREADS=12; python your_program.py`. It is recommen
ded to set it below 24.                                                                                              
  warnings.warn(                                                                                                     
WARNING:root:Failed to import geometry msgs in rigid_transformations.py.                                             
WARNING:root:Failed to import ros dependencies in rigid_transforms.py                                                
WARNING:root:autolab_core not installed as catkin package, RigidTransform ros methods will be unavailable            
Loading grasping labels...: 100%|████████████████████████████████████████████████████| 88/88 [01:50<00:00,  1.26s/it]
Loading data path and collision labels...: 100%|█████████████████████████████████████| 30/30 [00:09<00:00,  3.26it/s]
Loading data path and collision labels...: 100%|█████████████████████████████████████| 30/30 [00:04<00:00,  6.26it/s]
30                                                                                                                   
15 15                                                                                                                
start load object SDFs                                                                                               
100%|████████████████████████████████████████████████████████████████████████████████| 88/88 [00:04<00:00, 20.44it/s]
**** EPOCH 000 ****                                                                                                  
Current learning rate: 0.000040                                                                                      
Current BN decay momentum: 0.500000                                                                                  
2024-12-08 18:47:50.241533                                                                                           
/home/lidongjiang/anaconda3/envs/ldjpytorchlinux/lib/python3.9/site-packages/torch/nn/functional.py:4296: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_c
orners=True if the old behavior is desired. See the documentation of grid_sample for details.
  warnings.warn(
